src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,60 @@
+import classNames from 'classnames';
+import { useContext, useEffect, useState } from 'react';
+import { ErrorContext } from '../context/ErrorContext';
+
+enum ErrorMessage {
+  EmptyTitle = 'Title can`t be empty',
+}
```

Comment:
Actually you don't need to create an enum for 1 value, but you can leave it for the future improvements